frontend/app/routing.js

```diff
@@ -67,8 +67,51 @@ angular.module('openproject')
         }
       }
     })
+
+    .state('work-packages.show', {
+      url: '/work_packages/{workPackageId:[0-9]+}?query_props',
+      templateUrl: '/templates/work_packages.show.html',
+      controller: 'WorkPackageShowController',
+      controllerAs: 'vm',
+      resolve: {
+        workPackage: function(WorkPackageService, $stateParams) {
+          return WorkPackageService.getWorkPackage($stateParams.workPackageId);
+        },
+        // TODO hack, get rid of latestTab in ShowController
+        latestTab: function($state) {
+          var stateName = 'work-package.overview'; // the default tab
+
+          return {
+            getStateName: function() {
+              return stateName;
+            },
+            registerState: function() {
+              stateName = $state.current.name;
+            }
+          };
+        }
+      }
+    })
+    .state('work-packages.show.activity', {
+      url: '/activity',
+      templateUrl: '/templates/work_packages/tabs/activity.html'
+    })
+    .state('work-packages.show.activity.details', {
+      url: '#{activity_no:[0-9]+}',
+      templateUrl: '/templates/work_packages/tabs/activity.html'
+    })
+    .state('work-packages.show.relations', {
+      url: '/relations',
+      templateUrl: '/templates/work_packages/tabs/relations.html'
+    })
+    .state('work-packages.show.watchers', {
+      url: '/watchers',
+      controller: 'DetailsTabWatchersController',
+      templateUrl: '/templates/work_packages/tabs/watchers.html'
+    })
+
     .state('work-packages.list', {
-      url: '',
+      url: '/projects/:projectIdentifier/work_packages?query_id',
```

Comment:
For starters, there is a whole route dedicated to work_packages without a project scope: 

```
/work_packages
```
will show the complete list independent of a project. I will look into this a bit deeper tomorrow.

also, is the `query_id` part necessary? Arbitrary params should be accessible via `$params` or `$stateParams` (I mix those up all the time)